rtlib/rope.hh

```diff
@@ -29,6 +29,8 @@
 #include <algorithm>
 #include <cstring>
 #include <utility>
+#include <fstream>
+#include <sstream>
```

Comment:
do we need both for (de)serialization or is that to distinct the put methods?

Reply:
We need both just for the overloaded `put` method.